ViewModel/Saving/PersistenceManagers/FlowFrequencyPersistenceManager.cs

```diff
@@ -117,31 +118,28 @@ public string WriteFlowFrequencyToXML(AnalyticalFrequencyElement elem)
             flowFreqElem.SetAttributeValue(DESCRIPTION, elem.Description);
             flowFreqElem.SetAttributeValue(LAST_EDIT_DATE, elem.LastEditDate);
             flowFreqElem.SetAttributeValue(IS_ANALYTICAL, elem.IsAnalytical);
-            if(elem.IsAnalytical)
+            XElement analyticalElem = new XElement(ANALYTICAL_DATA);
+            flowFreqElem.Add(analyticalElem);
+            analyticalElem.SetAttributeValue(USES_MOMENTS, elem.IsStandard);
+            analyticalElem.SetAttributeValue(POR, elem.POR);
+            //if (elem.IsStandard)
```

Comment:
Looks like you decided to write everything out. That sounds good to me. Lets just remove this commented out code.

Reply:
reasonable.